Signal/translations/en.lproj/Localizable.strings

```diff
@@ -1411,6 +1411,9 @@
 /* Indicates the author of a quoted message. Embeds {{the author's name or phone number}}. */
 "QUOTED_REPLY_AUTHOR_INDICATOR_FORMAT" = "Replying to %@";
 
+/* Indicates the author of a quoted message is yourself. */
+"QUOTED_REPLY_AUTHOR_INDICATOR_YOURSELF" = "Yourself";
```

Comment:
I was thinking instead of "Replying to Alice" we'd say "Replying to Yourself", rather than just "Yourself". WDYT?

If we do go that route, it might still be good to have a separate, albeit redundant-in-english string like "Replying to Yourself" rather than trying to embed "Yourself" into `QUOTED_REPLY_AUTHOR_INDICATOR_FORMAT`. I'm not familiar enough with all our translations to know if that would work.

Reply:
Done.